molecule/default/prepare.yml

```
@@ -38,5 +38,16 @@
         'degraded' in systemctl_status.stdout
       when: ansible_service_mgr == "systemd"
 
+- name: >
+    Switch Debian platforms that are no longer supported to Debian
+    Archive package repositories
+  hosts: all
```

The files modified here are generally owned by `root` so I think this might be necessary.

```suggestion
  hosts: all
  become: true
  become_method: ansible.builtin.sudo
```